src/Cake.Issues.Markdownlint.Tests/LogFileFormat/MarkdownlintLogfileHelperTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace Cake.Issues.Markdownlint.Tests.LogFileFormat
+{
+    using Cake.Issues.Markdownlint.LogFileFormat;
+    using Cake.Issues.Testing;
+    using Shouldly;
+    using Xunit;
+
+    public sealed class MarkdownlintLogFileHelperTests
+    {
+        public sealed class TheCtor
```

Comment:
As this tests the `GetRegexForMarkdownIssueLine` the class should be name `TheGetRegexForMarkdownIssueLineMethod`